### DISCUSSION TITLE
Conditional Audit Skip

### DISCUSSION THREAD

**Discussion by aharrell-medsuite** [owen-it/laravel-auditing#958] [Q&A] [answered]
My issue is this, when certain fields are changed with specific values I do not require an audit record to be generated.  I have found several suggestions with shouldAudit() and isAuditableEvent() but neither of these stopped the audit.

Is there a way to do this?

**Comment by parallels999** [comment, marked as answer]
>when certain fields are changed with specific values I do not require an audit record to be generated

https://laravel-auditing.com/guide/audit-transformation.html
https://github.com/owen-it/laravel-auditing/blob/28ecd2d5cc05c3619f99af42611877f54371af20/config/audit.php#L104

Set `empty_values` to false, and use `transformAudit` and clean the audit on your specific values

Also 

https://laravel-auditing.com/guide/auditable-configuration.html#enable-disable
